package.json

```
@@ -4,6 +4,7 @@
   "version": "1.0.0",
   "license": "MIT",
   "main": "index.js",
```

The guide also recommends replacing `main` to `exports`.
```suggestion
  "exports": "./index.js",
```